includes/html/api_functions.inc.php

```diff
@@ -602,6 +602,39 @@ function get_bgp(Illuminate\Http\Request $request)
     return api_success($bgp_session, 'bgp_session');
 }
 
+function edit_bgp_descr(Illuminate\Http\Request $request)
+{
+    $data = json_decode($request->getContent(), true);
+    if (! $data) {
+        return api_error(500, 'Invalid JSON data');
+    }
+
+    $bgp = [];
+    //find existing bgp for update
+    $bgpPeerId = $request->route('id');
+    $bgps = dbFetchRows("SELECT * FROM `bgpPeers` WHERE `bgpPeer_id` = $bgpPeerId LIMIT 1");
```

Comment:
Hi @geg347 
Please have a look at the get_bgp function just above. A few checks are missing. You need to validate the data (if bgpPeerId is not numeric for instance) and most importantly the dbFetchRow must be injection_proof. Which is not the case here (using the question mark syntax).